File: Database/Models/Product.cs
using System.Data.SqlTypes;

namespace SweetShop.Database.Models
{
    public class Product
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        //ICollection<Category> Categories { get; set; }
        //ICollection<Image> Images { get; set; }
        //ICollection<Tag> Tags { get; set; }
    }
}

File: src/models/Product.js
class Product {
    constructor(id, name, description, price, image, categoryId, tags) {
      this.id = id;
      this.name = name;
      this.description = description;
      this.price = price;
      this.image = image;
      this.category = categoryId;
      this.tags = tags;
    }
}

export default Product;